pkg/apk/implementation.go

```diff
@@ -549,18 +549,26 @@ func (a *APK) FixateWorld(ctx context.Context, sourceDateEpoch *time.Time) error
 			return fmt.Errorf("cannot install due to conflict with %s", pkg)
 		}
 	}
+	allInstPkgs := make([]InstallablePackage, len(allpkgs))
+	for i, pkg := range allpkgs {
+		allInstPkgs[i] = pkg
+	}
+
+	return a.InstallPackages(ctx, sourceDateEpoch, allInstPkgs)
+}
 
+func (a *APK) InstallPackages(ctx context.Context, sourceDateEpoch *time.Time, pkgs []InstallablePackage) error {
```

Comment:
Would you mind either renaming this `pkgs` back to `allpkgs` or doing the `allpkgs` -> `pkgs` change as part of a separate commit? Just to make the diff and blame a little cleaner.